CMakeLists.txt

```diff
@@ -57,16 +64,21 @@ endforeach( OUTPUTCONFIG CMAKE_CONFIGURATION_TYPES )
 set(SC_MEMORY_SRC "${SC_MACHINE_ROOT}/sc-memory")
 
 if (${SC_USE_SANITIZER} STREQUAL "address")
-    set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=address -O1 -fno-omit-frame-pointer")
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address -O1 -fno-omit-frame-pointer")
+    set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=address -fno-omit-frame-pointer")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address -fno-omit-frame-pointer")
+
+    if(NOT SC_COVERAGE)
```

Comment:
Please do as was before. We don't want to run sanitizer and code coverage at one time.